oem/ibm/pfw-sms-utils/pfw_sms_menu.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef PFW_SMS_MENU_IBM_HPP
+#define PFW_SMS_MENU_IBM_HPP
+
+#include <string>
+
+namespace ibm_pfw_sms
+{
+
+/** @brief Return code from changePassword
+ *
+ * The "should never happen" codes should be captured for debug/diagnosis.
+ * Comments describe the conditions for which the other codes are used;
+ * these can be given to interactive users.
+ */
+enum changePasswordReasonCode
+{
+    PASSWORDCHANGESUCCESSFUL = 0x00,
```

Comment:
Could you please use same format as PAM_CONV_ERR?
adding underscore between words, it would help to align with other enum parameter and easy to read.

Reply:
Yes, done in latest patchset